internal/policies/mount/adsys_mount/src/main.rs

```diff
@@ -1,22 +1,52 @@
-use clap::Parser;
 use lib::AdsysMountError;
 mod logger;
 
-/// Arguments required to run this binary
-#[derive(Debug, clap::Parser)]
-#[command(version, about, long_about = None)]
-struct Args {
-    /// Path for the file containing the mounts for the user.
-    mounts_file: String,
-}
-
 fn main() -> Result<(), AdsysMountError> {
-    let args = Args::parse();
+    if std::env::args().any(|p| &p == "--help") {
+        print_help_msg();
+        return Ok(());
+    }
+
+    let mut args = std::env::args();
+
+    // Ignores the first argument, which is the path of the executable.
+    args.next();
+
+    let mounts_file = match args.next() {
+        Some(arg) => arg,
+        None => {
+            print_invalid_msg();
+            return Ok(());
```

Comment:
This is going to exit 0, non? It should rather return an error of invalid syntax (exit code 2)